### ISSUE TITLE
Problem: "Transfer Storage" storage service location type is missing to manage transfer Bagit packages 

### ISSUE THREAD

**Issue by peterVG** [archivematica/Issues#445]
As per issue #391, backlog transfers will be managed as Bagit packages by a new Storage Service in a new "Transfer Storage" storage service location that can respond to requests to list contents and download packages.  Bagit packages will be uncompressed.

---
**For Artefactual use:**
Please make sure these steps are taken before moving this issue from Review to Verified in Waffle:

- All PRs related to this issue are properly linked 👍
- All PRs related to this issue have been merged 👍
- Test plan for this issue has been implemented and passed 👍
- Documentation regarding this issue has been written and it has been added to the release notes, if needed 👍

blocked by #444

**Comment by sevein** [comment]
We've put this on hold since so far it's been possible to deal with both type of transfers (old transfers and baggited) within the same location.
